package TestCases;
import org.testng.Assert;
import org.testng.Reporter;
import org.testng.annotations.Test;

public class TestTryCatchScreenShot_11 {
	
	
	@Test(priority = 1)
	public void TesCaseTryCatchFailedForScreenShot()
	{
		
		try {
			Assert.fail("TesCaseTryCatchFailedForScreenShot Test Case is failed.");
		} catch (Exception e) {
			Reporter.log("<a target='_blank' href='D:\\Rajan\\Selenium-Practice\\19-TestNGLearning\\src\\test\\resources\\screenshots\\failure'><img src='D:\\Rajan\\Selenium-Practice\\19-TestNGLearning\\src\\test\\resources\\screenshots\\failure' height=50 width=50/></a>");
			/*
			 * Reporter.log("Steps to overcome fail test case"); Reporter.
			 * log("<a target=\"_blank\" href=\"F:\\screenshot\\error.jpg\">Screenshot link</a>"
			 * ); Reporter.log("<br>"); Reporter.
			 * log("<a target=\"_blank\" href=\"F:\\screenshot\\error.jpg\"><img height=200 width=200 src=\"F:\\screenshot\\error.jpg\"></a>"
			 * );
			 */
		}
		
	}

}
